jest.config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "moduleNameMapper": {
+    "@openmrs/esm-error-handling": "<rootDir>/src/__mocks__/openmrs-esm-error-handling.mock.tsx"
```

Comment:
You should mock the styleguide. Don't mock esm-error-handling. This will fix your error.

Reply:
I modified that and still the same error. I pushed the changes.

Reply:
Your description of your error isn't specific enough for me to diagnose the problem.

I am pulling down your fork and branch so that I can fix this myself.